src/types/requestArgs.ts

```diff
@@ -227,10 +227,15 @@ export interface CreateQueryParams extends BaseCRUDParams {
 
 export enum ColumnType {
```

Comment:
Might be good to add a reference to the docs here listing out the supported types:

```suggestion
// https://docs.dune.com/api-reference/tables/endpoint/create#body-schema-type
export enum ColumnType {
```